### DISCUSSION TITLE
Where is the implement of  the paper named "Making Smartphones Last Longer with Code Offload"?

### DISCUSSION THREAD

**Discussion by seimeicyx** [dotnet/maui#10613] [Q&A]
I am a graduate student，and I have read the paper named "MAUI: Making Smartphones Last Longer with Code Offload",but I can't find   the location of source code that implements this paper.
any one got help?

**Comment by tdeborde2** [comment]
I believe the MAUI referenced in this paper refers to the Mobile Assistance Using Infrastructure. This appears to be a research project and I am unable to find anything on it after 2015. Anyways, this project is for the Multi-platform application UI and is unrelated.
